src/tests/ftest/daos_test/rebuild.py

```diff
@@ -284,3 +284,19 @@ def test_rebuild_30(self):
         :avocado: tags=DaosCoreTestRebuild,daos_test,daos_core_test_rebuild,test_rebuild_30
         """
         self.run_subtest()
+
+    def test_rebuild_35(self):
+        """Jira ID: DAOS-16027
+
+        Test Description:
+            Run daos_test -r -s5 -u subtests=35
+
+        Use cases:
+            Core tests for daos_test rebuild
+
+        :avocado: tags=all,pr,daily_regression
```

Comment:
@phender and @daltonbohning I'm working on this separate patch due to DAOS-16016. The issue of extra time expenditure for new tests like this has come up in another PR https://github.com/daos-stack/daos/pull/14584

This one I estimate will run in about 2 minutes of execution time (based only on running with tmpfs on a local developer cluster, not functional hw testing in CI). Is it worth adding that to per-pr testing? I can see an argument for covering the new rebuild tests (particularly any that are very lengthy) to be tested only as part of daily_regression (not pr). It's still better than what we have now, which is no regular coverage for a few of these recently-developed tests. What do you think for this one (pr,daily_regression or just daily_regression)? We can take a look at how the hw functional testing shakes out too, to confirm my time estimate.

Reply:
It's not just the test time of a single test that's the issue. It's when we add 2 minutes here, 2 minutes there, ... and suddenly we have several hours of PR testing. For rebuild specifically, this is a huge issue. Last I checked, there is something like 6-8 hours of _**rebuild pr**_ testing. Which means even if a PR doesn't affect rebuild in any way, it still spends 6-8 hours running rebuild testing. (this is terrible for CI time and one of the contributors to slow turnaround times)

I can't tell you whether this test should be pr or daily or weekly because I don't know how likely it is to be affected by code changes. So instead, ask yourself
1. When pushing a PR, can you generally tell based on the code changes made that THIS test needs to be ran? And would using `Features: rebuild` be sensible?
2. How critical is this test? If this test fails, does it mean rebuild is effectively unusable? Or is it some edge case?

Realistically, we can't run everything in PR, so the idea is to mostly tag critical, quick tests with `pr`. Then, PRs should use `Features` to pull in additional test areas as needed. But at the same time, we don't have a perfect code <-> test mapping, so we don't always know whether we need to run a test.

Reply:
Yeah I think this one is a bit specific to reintegration and can be run in daily_regression, and in Features: rebuild in certain PRs, so we catch any inadvertent regressions in a fairly timely manner